### PR TITLE
docs(bms): fix docs issues.

### PR DESCRIPTION
**Pull request by wangzepeng666**
fix docs issues.

### 1、docs/resources/compute_bms_server_v2.md
add attribute `tenant_id` filed
image
image

### 2、docs/data-source/compute_bms_server_v2.md
add attribute `tenant_id` filed
image
image

### CODE REVIEW

File: docs/data-sources/compute_bms_nic_v2.md [hunk 1]
```diff
@@ -22,18 +22,29 @@ data "flexibleengine_compute_bms_nic_v2" "nic" {
...
+* `status` - (Optional, String) - The NIC port status.
+
+## Attribute Reference
 
 All of the argument attributes are also exported as result attributes.
 
 * `mac_address` - It is NIC's mac address.
 
 * `fixed_ips` - The NIC IP address.
+    The [fixed_ips](#<a name="functiongraph_fixed_ips"></a>) object structure is documented below.
```
Comment (ShiChangkuo) [inline docs/data-sources/compute_bms_nic_v2.md:41]:
```suggestion
    The [fixed_ips](#<a name="bms_nic_fixed_ips"></a>) object structure is documented below.
```

File: docs/data-sources/compute_bms_server_v2.md [hunk 2]
```diff
@@ -51,9 +54,8 @@ All of the argument attributes are also exported as result attributes.
 
 * `access_ip_v6` - This is a reserved attribute.  
 
-* `addresses` - It gives the BMS network address.
-
 * `security_groups` - The list of security groups to which the BMS belongs.
+    The [security_groups](#<a name="functiongraph_security_groups"></a>) object structure is documented below.
```
Comment (ShiChangkuo) [inline docs/data-sources/compute_bms_server_v2.md:58]: ditto

File: docs/resources/compute_bms_server_v2.md [hunk 3]
```diff
@@ -40,71 +40,75 @@ resource "flexibleengine_compute_bms_server_v2" "basic" {
...
-    creates a new bms server.
+* `region` - (Optional, String, ForceNew) The region in which to create the bms server instance. If
+    omitted, the `region` argument of the provider is used. Changing this will create a new bms server.
 
-* `name` - (Required) The name of the BMS.
+* `name` - (Required, String) The name of the BMS.
 
-* `image_id` - (Optional; Required if `image_name` is empty.) Changing this creates a new bms server.
+* `image_id` - (Optional, String, ForceNew) Changing this creates a new bms server. It is Required if `image_name` is
+  empty.
```
Comment (ShiChangkuo) [inline docs/resources/compute_bms_server_v2.md:49]:
```suggestion
* `image_id` - (Optional, String, ForceNew) It is Required if `image_name` is empty. Changing this creates a new bms server.
```

File: docs/resources/compute_bms_server_v2.md [hunk 4]
```diff
@@ -117,3 +121,20 @@ In addition to all arguments above, the following attributes are exported:
...
 
 * `host_status` - The nova-compute status: **UP, UNKNOWN, DOWN, MAINTENANCE** and **Null**.
+
+* `access_ip_v4` - This is a reserved attribute.
+
+* `access_ip_v6` - This is a reserved attribute.
+
+* `host_id` - Specifies the host ID of the BMS.
+
+* `tenant_id` - Specifies the ID of the tenant owning the BMS. The value is in UUID format.
```
Comment (ShiChangkuo) [inline docs/resources/compute_bms_server_v2.md:131]: `tenant_id` has been deprecated, it's not suggested to add it in docs.

File: docs/data-sources/compute_bms_nic_v2.md (hunk 1 above)
Comment (wangzepeng666) [reply docs/data-sources/compute_bms_nic_v2.md:41]: > 

fix

File: docs/resources/compute_bms_server_v2.md (hunk 3 above)
Comment (wangzepeng666) [reply docs/resources/compute_bms_server_v2.md:49]: > Changing this creates a new bms server.

fix

File: docs/resources/compute_bms_server_v2.md [hunk 5]
```diff
@@ -40,71 +40,75 @@ resource "flexibleengine_compute_bms_server_v2" "basic" {
...
+* `flavor_name` - (Optional, String) The name of the
     desired flavor for the bms server. Changing this resizes the existing bms server.
+    It is Required if `flavor_id` is empty.
 
-* `user_data` - (Optional) The user data to provide when launching the instance.
+* `user_data` - (Optional, String, ForceNew) The user data to provide when launching the instance.
     Changing this creates a new bms server.
 
-* `security_groups` - (Optional) An array of one or more security group names
+* `security_groups` - (Optional, Set) An array of one or more security group names
```
Comment (ShiChangkuo) [inline docs/resources/compute_bms_server_v2.md:65]: we use `List` for both TypeList and TypeSet
```suggestion
* `security_groups` - (Optional, List) An array of one or more security group names
```

**Pull request merged**